internal/http/handler/connection/info.go

```diff
@@ -1,22 +1,21 @@
-package RouteHandler
+package connection
 
 import (
 	"encoding/json"
 	"log"
 	"net/http"
 
-	"github.com/r-erema/wapi/src/Service/ConnectionsSupervisor"
-	session2 "github.com/r-erema/wapi/src/Session"
-
 	"github.com/Rhymen/go-whatsapp"
 	"github.com/gorilla/mux"
+	"github.com/r-erema/wapi/internal/model/session"
+	"github.com/r-erema/wapi/internal/service/supervisor"
 )
 
 type GetActiveConnectionInfoHandler struct {
-	connectionSupervisor ConnectionsSupervisor.Interface
+	connectionSupervisor supervisor.ConnectionSupervisor
```

Comment:
If it is possible, make the name of supervisor.ConnectionSupervisor a bit shorter. supervisor.Connection?